.github/workflows/nm-remote-push.yml

```diff
@@ -37,7 +37,7 @@ jobs:
             test_label_solo: gcp-k8s-l4-solo
             test_label_multi: ignore
             test_timeout: 480
-            test_skip_list: neuralmagic/tests/skip-for-remote-push-tmp.txt
+            test_skip_env_vars: neuralmagic/tests/test_skip_env_vars/smoke.txt
```

Comment:
For remote-push, maybe we don't need to run all 4 python versions, and only 38/311 is good enough?

Reply:
yeah, let's get to that after this is merged